src/gsi/read_satwnd.f90

```diff
@@ -637,6 +678,7 @@ subroutine read_satwnd(nread,ndata,nodata,infile,obstype,lunout,gstime,twind,sis
            ee=r110
            qifn=r110
            qify=r110
+           qm=2
```

Comment:
Is there a reason why `qm` is set to 2 here when it is also set to 2 on Line 727?

Reply:
This is the better place to set qm=2, it's a "blanket" initialization in case an AMV comes without a qm value (GOES winds no longer carry qm ). The later occurrence was probably put in just in case (better safe than sorry).